### PR TITLE
fixed the description of the role

### PR DESCRIPTION
**Pull request by BeReasonable**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: ProjectWeb/project-for-the-web-security-roles.md [hunk 1]
```diff
@@ -27,11 +27,15 @@ When a project is shared with an AAD Office Group, the Microsoft Project Applica
...
 ## Project Common
-- Cannot be customized
+- Customizable
 - Provides non-project related permissions to give a user access to the environment including the ability to log in. 
 - It is a copy of the Common Data Service User role but can diverge in future releases.
 - The AAD Office Group team that is created when a project is shared is given this role so that members have enough permissions to log into and interact with the environment
 
+## Project User
+- Cannot be customized
+- Provides user scoped permissions to create, read, update, and delete project and related entities.
```
Comment (JohanFreelancer9) [inline ProjectWeb/project-for-the-web-security-roles.md:37]:
```suggestion
- Provides user scoped permissions to create, read, update, and delete projects and related entities. 
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by BeReasonable** [comment]
Hi @JohanFreelancer9 , are there any changes I need to make to have this pr merged?

**Comment by JohanFreelancer9** [comment]
@BeReasonable - from an editing point it is all good, thanks. @scanum?

**Comment by BeReasonable** [comment]
This section was already updated.

**Pull request closed without merging**